### PR TITLE
Kops - Add presubmits for network plugins

### PR DESCRIPTION
**Pull request by hakman**
/cc @rifelpet

### CODE REVIEW

File: config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml [hunk 1]
```diff
@@ -0,0 +1,314 @@
+presubmits:
+  kubernetes/kops:
+
+  - name: pull-kops-e2e-cni-amazonvpc
+    branches:
+    - master
+    run_if_changed: '^upup\/models\/cloudup\/resources\/addons\/networking\.amazon-vpc-routed-eni\/'
+    skip_report: false
```
Comment (rifelpet) [inline config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml:8]: can we make these `optional: true` to start? that way we dont bombard all the open PRs with test failures if theres an issue with their setup.

Comment (hakman) [reply config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml:8]: it will affect only those with the path and there are none at the moment

Comment (rifelpet) [reply config/jobs/kubernetes/kops/kops-presubmits-network-plugins.yaml:8]: ah ok, that sounds good

**Pull request merged**